src/elm/Page/Shop.elm

```diff
@@ -351,12 +347,14 @@ viewCard model ({ shared } as loggedIn) index card =
                 ]
             ]
         , div
-            [ class "hidden md:visible md:flex md:flex-wrap rounded-lg hover:shadow-lg bg-white overflow-hidden"
+            [ class "hidden md:visible md:flex md:flex-wrap rounded-lg hover:shadow-lg bg-white"
             ]
-            [ div [ class "w-full relative bg-gray-500" ]
-                [ img [ class "w-full h-48 object-cover", src image ] []
+            [ div [ class "w-full relative bg-gray-500 rounded-t-lg" ]
+                [ img [ class "w-full h-48 object-cover rounded-t-lg", src image ] []
                 , div [ class "absolute right-1 bottom-1 " ]
-                    [ Profile.view shared loggedIn.accountName card.product.creator ]
+                    [ Profile.Summary.view loggedIn.shared loggedIn.accountName card.product.creator card.profileSummary
```

Comment:
It feels kind of hacky, because we're using nested `a` tags, and apparently [nested links are illegal](https://www.w3.org/TR/html401/struct/links.html#h-12.2.2), but we can just `preventDefault` and `stopPropagation` on click!

Reply:
I like it and it works well